### DISCUSSION TITLE
Reframework that works with resident evil revelations.

### DISCUSSION THREAD

**Discussion by avnpa** [praydog/REFramework#1079] [General Questions] [answered]
I am currently trying to play resident evil revelations and using cheat happens trainer.  As soon as I try to activate the trainer my game crashes.  I looked on the cheat happens site and Caliber mentioned using the RE4 reframe.  Still crashing am I doing something wrong?  Thanks

**Comment by praydog** [comment, marked as answer]
Not RE Engine.
